src/bundle/Resources/config/routing.yml

```diff
@@ -8,3 +8,8 @@ ez_hybrid_platform_ui_version_draft_actions:
     defaults:
         _controller: EzSystemsHybridPlatformUiBundle:Version:draftActions
     methods: [POST]
+ez_hybrid_platform_ui_location_actions:
+    path: /location/actions/{contentId}/{locationId}
```

Comment:
Could you please change that to `/content/{contentId}/locations` ?
`POST` is enough to indicate the `action` part I think.